src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    method="post"
+    class="input-form"
+    data-qa="big"
+    >
```

Comment:
```suggestion
    <form
      action="#"
      method="post"
      class="input-form"
      data-qa="big"
    >
```
fix the formatting everywhere